programs/nft-voter/src/state/registrar.rs

```diff
@@ -0,0 +1,97 @@
+use crate::{
+    error::NftVoterError,
+    id,
+    state::CollectionConfig,
+    tools::{spl_token::get_spl_token_amount, token_metadata::get_token_metadata_for_mint},
+};
+use anchor_lang::prelude::*;
+use spl_governance::tools::spl_token::{get_spl_token_mint, get_spl_token_owner};
+
+/// Registrar which stores NFT voting configuration for the given Realm
+#[account]
+#[derive(Debug, PartialEq)]
+pub struct Registrar {
+    /// spl-governance program the Realm belongs to
+    pub governance_program_id: Pubkey,
```

Comment:
nitpick: I see that you are not storing the bump, maybe its fine, as long as your dont need to sign with the registrar down the line